### DISCUSSION TITLE
need help to get websocket close reason code ?

### DISCUSSION THREAD

**Discussion by calvin2021y** [curl/curl#10556] [Q&A]
Is there a way to get websocket close reason code (and message)?

**Comment by calvin2021y** [comment]
by parse data from write callback, with meta flags  get the code reason.

**Reply by bagder** [reply]
If that doesn't work (good enough) or you have additional ideas or suggestions, feel free to let us know. We are still working on improving the websockets API.

**Reply by calvin2021y** [reply]
It work. read first 2 byte form message will get the reason code, the rest is the message.
